openquake/qa_tests_data/scenario_risk/case_shakemap/pre-job.ini

```diff
@@ -3,3 +3,4 @@ description = Peru
 calculation_mode = scenario
 exposure_file = exposure_models/exposure_model-per_com.xml
 structural_vulnerability_file = IndComm-structural_vulnerability_model.xml
+#structural_fragility_file = structural_frag.xml
```

Comment:
Please check if this has to remain commented

Reply:
For the moment yes.